client/src/ui-components/ShoppingCartComponent/ModalOrderConfirmed.js

```diff
@@ -0,0 +1,37 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./ShoppingCartControlsCSS.css";
+import React from "react";
+import { Button, Modal } from "react-bootstrap";
+
+function ModalOrderConfirmed(props) {
+
+  return (
+    <Modal show={props.show} onHide={props.handleClose}>
+      <Modal.Header closeButton>
+        <Modal.Title className="modal-title">Are you sure?</Modal.Title>
+      </Modal.Header>
+      <ul>
+        <li>Output map to send to createOrder:</li>
+        {Array.from(props.cart.entries()).map((entry) => {
+          return (
+            <li>
+              {entry[0]}
+              {"->"}
+              {entry[1]}
+            </li>
+          );
+        })}
```

Comment:
Not user friendly: use instead the product's name
"output map to send to createOrder" is also not very user friendly